### DISCUSSION TITLE
Add multiple IMAP accounts to each user or just to one user and then grant rights ?

### DISCUSSION THREAD

**Discussion by computersalat** [Intermesh/groupoffice#1071] [General]
Hi,

did an installation for a customer with 5 user. Each user should have access to different IMAP postboxes.
GroupOffice is a great tool enabling you to add multiple IMAP postboxes for each user.
While I was playing around I noticed two ways of giving users access to the postboxes.
solution A)
- Add all IMAP accounts the user that he/she should have access to

solution B)
- add all IMAP accounts to one user and grant access to those postboxes the user should be able to access.

With solution A I see multiple postboxes of one postbox, e.g. info@domain.com, but with different Owner.
What is the difference ?
What is the better way ?

**Comment by computersalat** [comment]
currently I see one difference:
- With solution A) every user is able to configure the postbox (e.g. signature) on his own. It needs to be done as often as the postbox is added to a user.
- With soliution B) I guess, e.g. the signature just needs to be configured once. Every user that was granted access gets the same information.

Are there more differences I do not see yet ?

**Comment by mschering** [comment]
Both ways are valid but I'd prefer B as you only have to maintain one account.

**Reply by computersalat** [reply]
Yes, IMHO the better way. Will change it.
Thank you
